WalletWasabi.Gui/Controls/WalletExplorer/SendTabViewModel.cs

```diff
@@ -275,6 +277,25 @@ public SendTabViewModel(WalletViewModel walletViewModel, bool isTransactionBuild
 							return;
 						}
 					}
+
+					decimal customFee = 0;
+					if (IsCustomFee)
+					{
+						if (decimal.TryParse(UserFeeText.Replace(',', '.'), out customFee))
```

Comment:
This is dangerous. I strongly recommend to use `Money` instead of `decimal` here.

Reply:
Can you do something like 3.4 satoshis with Money

Reply:
Wow, sorry, this is not the absolute fee but the fee rate instead, I got it. In that case you could use the `FeeRate` class. I think it has a constructor that receives money anyway.